flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionCapacitySchedulerITCase.java

```diff
@@ -284,7 +284,7 @@ public void testTaskManagerFailure() throws Exception {
 			ConcurrentMap<ContainerId, Container> containers = nm.getNMContext().getContainers();
 			for (Map.Entry<ContainerId, Container> entry : containers.entrySet()) {
 				String command = StringUtils.join(entry.getValue().getLaunchContext().getCommands(), " ");
-				if (command.contains(YarnTaskManager.class.getSimpleName())) {
```

Comment:
The rework of this test is handled by [FLINK-11348](https://issues.apache.org/jira/browse/FLINK-11348). Currently it is ignored. We simply make this change in order not to block the removal of legacy yarn components.